packages/web3torrent/.storybook/web3torrent-client.mock.js

```diff
@@ -1,3 +1,22 @@
+const React = require('react');
+const utils = require('./../src/utils/test-utils')
```

Comment:
are these files linted? There is a missing semicolon at the end of the line.

Reply:
uhmm... for some reason, the linter is ignoring these files, even when they are not in the .eslintignore, neither the local nor the global one.
I linted the files now

Reply:
FYI, my VSCode editor adds the missing semicolon on save via the Prettier plugin.

Reply:
I think I have the onSave deactivated :-P